crates/server/src/admin/utils/auth.rs

```diff
@@ -15,10 +16,11 @@ use tracing::info;
 use crate::admin::service::{parse_api_error, ApiError};
 use crate::admin::storage::root_key::get_root_key;
 use crate::verifysignature::verify_near_signature;
+use crate::verifysnsignature::{verify_argent_signature, verify_metamask_signature};
 
 pub fn verify_node_signature(
```

Comment:
_...

```suggestion
pub async fn verify_node_signature(
```

Reply:
For this, the thing is that currently, all parent functions are not async, this is why I implemented it in this way. The reason is that I didn't want to change the current implementation of all other wallets.